qa/suites/upgrade/reef-p2p/reef-p2p-stress-split/objectstore/filestore-xfs.yaml

```diff
@@ -0,0 +1,15 @@
+overrides:
```

Comment:
On your latest runs, a test failed with Filestore as the objectstore. Here is the reason for failure:
```
2024-02-24T06:13:28.050 DEBUG:teuthology.orchestra.run.smithi063:> sudo mount -t xfs -o noatime /dev/vg_nvme/lv_4 /var/lib/ceph/osd/ceph-3
2024-02-24T06:13:28.102 DEBUG:teuthology.orchestra.run.smithi063:> sudo /sbin/restorecon /var/lib/ceph/osd/ceph-3
2024-02-24T06:13:28.114 INFO:teuthology.orchestra.run.smithi063.stderr:sudo: /sbin/restorecon: command not found
2024-02-24T06:13:28.115 DEBUG:teuthology.orchestra.run:got remote process result: 1
2024-02-24T06:13:28.115 DEBUG:teuthology.orchestra.run.smithi063:> sudo MALLOC_CHECK_=3 adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage ceph-osd --no-mon-config --cluster ceph --mkfs --mkkey -i 0 --monmap /home/ubuntu/cephtest/ceph.monmap
2024-02-24T06:13:28.189 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.183+0000 7fcd11b5e700 -1 WARNING: all dangerous and experimental features are enabled.
2024-02-24T06:13:28.189 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.183+0000 7fcd11b5e700 -1 WARNING: all dangerous and experimental features are enabled.
2024-02-24T06:13:28.190 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.187+0000 7fcd11b5e700 -1 WARNING: all dangerous and experimental features are enabled.
2024-02-24T06:13:28.190 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.187+0000 7fcd11b5e700 -1 create: FileStore has been deprecated and is no longer supported
2024-02-24T06:13:28.191 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.187+0000 7fcd11b5e700 -1 unable to create object store
2024-02-24T06:13:28.198 DEBUG:teuthology.orchestra.run:got remote process result: 250
2024-02-24T06:13:28.198 DEBUG:teuthology.orchestra.run.smithi063:> sudo MALLOC_CHECK_=3 adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage ceph-osd --cluster ceph --mkfs --mkkey -i 0 --monmap /home/ubuntu/cephtest/ceph.monmap
2024-02-24T06:13:28.239 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.235+0000 7fbaa13ff700 -1 WARNING: all dangerous and experimental features are enabled.
2024-02-24T06:13:28.239 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.235+0000 7fbaa13ff700 -1 auth: unable to find a keyring on /var/lib/ceph/osd/ceph-0/keyring: (2) No such file or directory
2024-02-24T06:13:28.239 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.235+0000 7fbaa13ff700 -1 AuthRegistry(0x56218e06a140) no keyring found at /var/lib/ceph/osd/ceph-0/keyring, disabling cephx
2024-02-24T06:13:28.239 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.235+0000 7fbaa13ff700 -1 auth: unable to find a keyring on /var/lib/ceph/osd/ceph-0/keyring: (2) No such file or directory
2024-02-24T06:13:28.240 INFO:teuthology.orchestra.run.smithi063.stderr:2024-02-24T06:13:28.235+0000 7fbaa13ff700 -1 AuthRegistry(0x7ffe3f86cbb0) no keyring found at /var/lib/ceph/osd/ceph-0/keyring, disabling cephx
2024-02-24T06:13:28.240 INFO:teuthology.orchestra.run.smithi063.stderr:failed to fetch mon config (--no-mon-config to skip)
2024-02-24T06:13:28.244 DEBUG:teuthology.orchestra.run:got remote process result: 1
2024-02-24T06:13:28.245 ERROR:teuthology.contextutil:Saw exception from nested tasks
Traceback (most recent call last):
  File "/home/teuthworker/src/github.com_ceph_ceph_9752e83ac52d3c9a1eb27ac162efa780cafb82d0/qa/tasks/ceph.py", line 999, in cluster
    remote.run(args=args)
  File "/home/teuthworker/src/git.ceph.com_teuthology_b1dac5519c57c269ea98a22fb7729016a1d0e4d2/teuthology/orchestra/remote.py", line 523, in run
    r = self._runner(client=self.ssh, name=self.shortname, **kwargs)
  File "/home/teuthworker/src/git.ceph.com_teuthology_b1dac5519c57c269ea98a22fb7729016a1d0e4d2/teuthology/orchestra/run.py", line 455, in run
    r.wait()
  File "/home/teuthworker/src/git.ceph.com_teuthology_b1dac5519c57c269ea98a22fb7729016a1d0e4d2/teuthology/orchestra/run.py", line 161, in wait
    self._raise_for_status()
  File "/home/teuthworker/src/git.ceph.com_teuthology_b1dac5519c57c269ea98a22fb7729016a1d0e4d2/teuthology/orchestra/run.py", line 181, in _raise_for_status
    raise CommandFailedError(
teuthology.exceptions.CommandFailedError: Command failed on smithi063 with status 250: 'sudo MALLOC_CHECK_=3 adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage ceph-osd --no-mon-config --cluster ceph --mkfs --mkkey -i 0 --monmap /home/ubuntu/cephtest/ceph.monmap'
```

Since FileStore is depreciated, we should remove this yaml.